ee/clickhouse/sql/retention/retention.py

```diff
@@ -15,6 +15,7 @@
     ORDER BY 
         breakdown_values, 
         intervals_from_base
+    SETTINGS timeout_before_checking_execution_speed = 60
```

Comment:
Let's not pass settings this way - instead use the `settings` arg of sync_execute.